awx_collection/plugins/modules/tower_notification.py

```diff
@@ -196,22 +216,31 @@
       default: "present"
       choices: ["present", "absent"]
       type: str
-
-requirements:
-- ansible-tower-cli >= 3.0.2
-
+    tower_oauthtoken:
+      description:
+        - The Tower OAuth token to use.
+      version_added: "3.7"
+      type: str
 extends_documentation_fragment: awx.awx.auth
 '''
 
 
 EXAMPLES = '''
-- name: Add Slack notification
+- name: Add Slack notification with custom messages
   tower_notification:
     name: slack notification
+    organization: Default
     notification_type: slack
     channels:
       - general
```

Comment:
I think the example shouldn't have `channels`, that's mixed messaging.

```
[DEPRECATION WARNING]: channels parameter has been depricated, please use notification_configuration instead.. This feature will be removed in version 3.6. Deprecation warnings can
 be disabled by setting deprecation_warnings=False in ansible.cfg.
```

Reply:
All documentation updated to use notification_configuration.